packages/rspeedy/core/src/plugins/chunkLoading.plugin.ts

```diff
@@ -7,36 +7,23 @@ import type { RsbuildPlugin } from '@rsbuild/core'
 import { LynxCacheEventsPlugin } from '@lynx-js/cache-events-webpack-plugin'
 import { ChunkLoadingWebpackPlugin } from '@lynx-js/chunk-loading-webpack-plugin'
 
-import { isLynx } from '../utils/is-lynx.js'
-import { isWeb } from '../utils/is-web.js'
-
 export function pluginChunkLoading(): RsbuildPlugin {
   return {
     name: 'lynx:rsbuild:chunk-loading',
     setup(api) {
-      api.modifyBundlerChain((chain, { environment }) => {
-        if (isWeb(environment)) {
-          chain
-            .output
-            .chunkLoading('import-scripts')
-            .end()
-          return
-        }
-
-        if (isLynx(environment)) {
-          // dprint-ignore
-          chain
-          .plugin('lynx:chunk-loading')
-            .use(ChunkLoadingWebpackPlugin)
-          .end()
-          .plugin('lynx:cache-events')
-            .use(LynxCacheEventsPlugin)
-          .end()
-          .output
-            .chunkLoading('lynx')
-            .chunkFormat('commonjs')
-          .end()
-        }
+      api.modifyBundlerChain((chain) => {
+        // dprint-ignore
+        chain
+        .plugin('lynx:chunk-loading')
+          .use(ChunkLoadingWebpackPlugin)
+        .end()
+        .plugin('lynx:cache-events')
+          .use(LynxCacheEventsPlugin)
+        .end()
+        .output
+          .chunkLoading('lynx')
```

Comment:
Like I said in https://github.com/lynx-family/lynx-stack/pull/1677#discussion_r2340797870. I think it would be better to make a change in the plugin.